api_includes.ts

```diff
@@ -80,7 +80,9 @@ type UnwrapData<Data, Include extends IncludeParameter> =
     // Each item of a data array has to be unwrapped individually (except primitives).
     : Data extends Array<infer Item>
     // Turn off distributivity to leave primitive union types alone.
-      ? [Item] extends [object] ? WithIncludes<Item, Include>[] : Data
+      ? [Item] extends [string] ? Item[]
+      : [Item] extends [number] ? Item[]
+      : Item extends object ? WithIncludes<Item, Include>[] : Data
```

Comment:
```suggestion
      ? [Item] extends [string | number | undefined] ? Item[]
      : Item extends object ? WithIncludes<Item, Include>[]
      : Item[]
```
We can combine the two cases here and also add `undefined` in case a union type includes it.
Cool that it is finally possible to state the type as `Item[]` which is clearer than just `Data`, I think this was not possible before turning off the distributivity.
While I don't always agree with the formatters behavior for these complex types (and sneaked in some comments at the right spot to force line breaks where I want to have them), it makes sense to me to have the fallback case (which should not happen for our use case) on its own line.